### ISSUE TITLE
php8.1 throws deprecation message for strcasecmp()

### ISSUE THREAD

**Issue by martin-rueegg** [humhub/cfiles#165]
When trying to select the "Files" entry in a Space Menu, you'll get a PHP deprecation message when running on PHP 8.1 or higher.

```
strcasecmp(): Passing null to parameter #1 ($string1) of type string is deprecated
```

The error is thrown in https://github.com/yiisoft/yii2/blob/master/framework/helpers/BaseHtml.php#L345 but the culprit is the call to `Html::beginForm()` in https://github.com/humhub/cfiles/blob/7c6b224c039c40996c23fd5a2efb1f80a154b530/views/browse/index.php#L29, which uses `null` rather than `'post'` as a second argument.

The issue has already been fixed in https://github.com/humhub/cfiles/pull/159 but there has been no new release since, which is unfortunate.

So this issue is for documentation purposes and hoping to get the fix to be released.

**Comment by yurabakhtin** [comment]
@luke- 

> The issue has already been fixed in https://github.com/humhub/cfiles/pull/159 but there has been no new release since, which is unfortunate.

Could you please do new release?

**Comment by luke-** [comment]
Just released 0.15.0

**Comment by martin-rueegg** [comment]
Thanks, @luke- 

PS: Error doesn't happen anymore after updating.
